docs/config.rst

```diff
@@ -1,11 +1,502 @@
+
+
+Add a Flow Step
+^^^^^^^^^^^^^^^
+If you want to add a step to a flow, you first need to know what the existing steps are.
+The easiest way to see this is with ``cci flow info <flow_name>``.
+Take the ``dev_org`` flow as an example:
+
+.. code-block:: console
+
+    $ cci flow info dev_org
+    Description: Set up an org as a development environment for unmanaged metadata
+    1) flow: dependencies [from current folder]
+        1) task: update_dependencies
+        2) task: deploy_pre
+    2) flow: deploy_unmanaged
+        0) task: dx_convert_from
+        when: project_config.project__source_format == "sfdx" and not org_config.scratch
+        1) task: unschedule_apex
+        2) task: update_package_xml
+        when: project_config.project__source_format != "sfdx" or not org_config.scratch
+        3) task: deploy
+        when: project_config.project__source_format != "sfdx" or not org_config.scratch
+        3.1) task: dx_push
+            when: project_config.project__source_format == "sfdx" and org_config.scratch
+        4) task: uninstall_packaged_incremental
+        when: project_config.project__source_format != "sfdx" or not org_config.scratch
+    3) flow: config_dev
+        1) task: deploy_post
+        2) task: update_admin_profile
+    4) task: snapshot_changes
+
+There are four steps to this flow. The first three steps are themselves flows, and the last step is a task.
+
+All **non-negative numbers and decimals** are valid as step numbers in a flow.
+This makes it easy to add steps before, in-between, or after existing flow steps.
+If we wanted to add a step at the beginning of the dev org flow, valid step numbers would include any number greater than or equal to zero and less than 1.
+Example values would include: 0, 0.3, and 0.89334.
+All of these would cause the step to execute before step 1 in the ``dev_org`` flow.
+
+If you wanted to add a step between steps 2 and 3, then a step number of 2.5 could be utilized.
+
+If you wanted to add a step number that runs after all steps in the flow, then any step number greater than 4 could be utilized.
+
+
+
+Skip a Flow Step
+^^^^^^^^^^^^^^^^
+To skip a flow step, set the desired step number to a task with the value of ``None``.
+The following would skip the 4th step from the ``dev_org`` flow.
+
+.. code-block:: yaml
+
+    dev_org:
+        steps:
+            4:
+                task: None
+
+When CumulusCI detects a task with this value, it is skipped:
+
+.. image:: images/skipping_task.png
+
+
+
+Replace a Flow Step
+^^^^^^^^^^^^^^^^^^^
+To replace a flow step, simply name the task or flow you wish to run in place of the current step.
+The following would replace the fourth step of the ``dev_org`` flow with a custom task that loads data into a dev environment.
+
+.. code-block:: yaml
+
+    dev_org:
+        steps:
+            4:
+                task: load_data_dev
+
+You can swap two steps in a flow by replacing one with the other. 
```

Comment:
If the existing step is a task and you're switching to a flow, you also have to set `task: None` so that the merged step config doesn't reference both a task and a flow. And vice versa if you're replacing a flow with a task.